sdks/go/pkg/beam/core/runtime/exec/sdf_test.go

```diff
@@ -925,3 +1083,105 @@ func (rt *SplittableUnitRTracker) TrySplit(_ float64) (interface{}, interface{},
 func (rt *SplittableUnitRTracker) GetProgress() (float64, float64) {
 	return rt.Done, rt.Remaining
 }
+
+// TestMultiWindowProcessing tests that ProcessSizedElementsAndRestrictions
+// handles processing multiple windows correctly, even when progress is
+// reported and splits are performed during processing.
+func TestSplittingValidationProcessing(t *testing.T) {
```

Comment:
This function looks like just a copy of TestMultiWindowProcessing. I'm guessing you were going to modify it to test the invalid split size and either forgot or changed your mind?

Regardless, I think the tests you currently have provide enough coverage, so you should remove this function.

Reply:
Your right, thanks, will remove.